addon/ng2/blueprints/ng2/files/src/app.scss

```diff
@@ -0,0 +1,6 @@
+// --------------------------------
+// Import all your other sass files below
+// Note: Order counts, so import your variables and mixins first
+// --------------------------------
+
+// e.g. @import 'app/components/example/example.scss'
```

Comment:
listing every single scss file here is too error prone and besides, angular compiler will fetch all css files and prefix all selectors by default to provide css isolation. so we definitely don't want to list individual files here.